### DISCUSSION TITLE
How to require chai in nodejs project?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [chaijs/chai#1630] [Support] [answered]
How to require chai in nodejs project? 

I find there is [require way](https://github.com/tree-sitter/node-tree-sitter/blob/master/test/tree_test.js#L4) to use chai,

but when I use require in nodejs project, I get error:
```
const { assert } = require("chai");
                   ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/mark/Desktop/code/Test/Test-demos/nodejs/nodejs-node-tree-sitter-test-01/node_modules/.pnpm/chai@5.1.1/node_modules/chai/chai.js from /Users/mark/Desktop/code/Test/Test-demos/nodejs/nodejs-node-tree-sitter-test-01/test/tree_test.js not supported.
```

**Comment by 43081j** [comment, marked as answer]
Chai 5 is esm only

There are various GitHub discussions about this if you search.

As mentioned in the other discussions and issues: 

- stick with chai 4
- move to esm
- use a dynamic import

One of the above is the solution
